MapleServer2/Types/Player.cs

```diff
@@ -152,6 +153,11 @@ public SubJobCode SubJobCode
     public Wallet Wallet { get; set; }
     public Dictionary<int, QuestStatus> QuestData;
 
+    public Dictionary<int, Shop> Shops = new();
+    public Dictionary<int, PlayerShopLog> ShopLogs = new();
+    public Dictionary<int, PlayerShopItemLog> ShopItemLogs = new();
```

Comment:
Why we are storing logs in the player? 🤔

Reply:
I figured this is easier than hitting the database everytime someone opens a shop up.